DEVELOPMENT.md

```diff
@@ -16,3 +23,30 @@ You can run the unit tests by simply doing
 ```bash
 make test
```

Comment:
Also a note here about running this prior to committing any changes, especially in the situation when API changes are made.